packages/runtime/src/state/react-builder-preview.ts

```diff
@@ -759,12 +814,13 @@ export function configureStore({
   client,
 }: {
   rootElements?: Map<string, Documents.Element>
-  preloadedState?: PreloadedState<State>
```

Comment:
`PreloadedState`, which adds recursive optionality to the passed type, doesn't work on `Map`s, but `Partial` is just fine here as we don't need recursive optionality anyway.